nearcore/src/config.rs

```diff
@@ -1007,6 +1010,21 @@ pub fn init_configs(
             info!(target: "near", "Generated node key, validator key, genesis file in {}", dir.display());
         }
     }
+
+    if let Some(start_protocol_version) = dump_epoch_config {
+        let epoch_config_dir = dir.join("epoch_configs");
+        fs::create_dir_all(epoch_config_dir.clone())
+            .with_context(|| anyhow!("Failed to create directory {:?}", epoch_config_dir))?;
+        EpochConfigStore::for_chain_id(MAINNET, None)
```

Comment:
can we do this by matching chain_id to mainnet or testnet (ie. use testnet configs if chain_id == TESTNET), and also give error if chain-id is not one of mainnet or testnet?

Reply:
I see what you mean. In case we fork from testnet, we would want to dump and amend the testnet epoch config. 
There are other uses of this tool where you can bring your own image that is not mainnet or testnet. In that case you would not pass a `chain_id` to the mirror tool. In this case we would need to falback to a default value, i.e. `mainnet`.

For now, we do not plan to use any testnet forknet so I added this to the backlog in the Forknet tracker issue.